app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/PlanWizardInstancePropertiesStepSelectors.js

```diff
@@ -0,0 +1,33 @@
+import { OSP_TENANT, EMS_CLUSTER } from '../../../../OverviewConstants';
```

Comment:
Asking a question, TOTALLY A QUESTION could this be renamed to be a `helpers.js`?  `PlanWizardInstancePropertiesStepSelectors.js ` is kinda ah mouthful 😟

Reply:
I think you're right. I'll rename this one.